.travis.yml

```diff
@@ -1,20 +1,26 @@
 language: node_js
 
-before_script:
- - sh -e /etc/init.d/xvfb start
-
-install: ant
-
 env:
- - DISPLAY=':99.0'
+  global:
+    - DISPLAY=':99.0'
+  matrix:
+    - MOZDOWNLOADPARAMS='-t daily'
+    - MOZDOWNLOADPARAMS='-t daily --branch=mozilla-aurora'
+    - MOZDOWNLOADPARAMS='-t release -v latest-beta'
+    - MOZDOWNLOADPARAMS='-t release -v latest'
+    - MOZDOWNLOADPARAMS='-t release -v latest-esr'
```

Comment:
> Didn't we wanted to skip tests for ESR31?

We talked about it, but I didn't see the point why just ESR testing should be skipped.
I vote for including because
 * all failing test was #208 
 * not just ESR, but Aurora and Beta also had failed build in this PR
 * last build was successful for all tested version of Firefox

So I kept ESR31 testing.